.gitignore

```diff
@@ -32,6 +34,7 @@ playbooks/*.retry
 
 .vagrant/
 Vagrantfile.local
+.secrets.json
```

Comment:
if the `.secret*` is in the ignore, why do we need this one as well?

Reply:
agree. cleaning it up.